db/migrate/20250303154718_increase_report_ticket_url_limit.rb

```diff
@@ -0,0 +1,7 @@
+class IncreaseReportTicketUrlLimit < ActiveRecord::Migration[7.1]
+  uses_departure! if Rails.env.staging? || Rails.env.production?
+
+  def change
```

Comment:
According to https://guides.rubyonrails.org/active_record_migrations.html#changing-columns we need to use #up and #down for a reversible change_column migration.

Reply:
Sorry, I forgot to add a note: I don't think we should make a down migration, because there is a risk of data loss if limits are changed to a lower length

Reply:
Let's make that clear by moving `change_column` to `up` and have a `down` method that is either empty or throwing [an irreversible error](https://guides.rubyonrails.org/active_record_migrations.html#throwing-an-error-to-prevent-reverts).